info_processing/code_processing/treesitter.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+from collections import Counter
+from typing import Dict
+
+from dulwich.porcelain import clone
+from dulwich.repo import Repo
+
+from pathlib2 import Path
+
+from tree_sitter import Language, Parser
+
+parser = Parser()
+logger = logging.getLogger(__name__)
+
+
+def setup_tree_sitter_parser() -> None:
+    """
+    Function sets up the parser for Python, JavaScript and Java languages cloning the repositories, that contain
+    grammars.
+    :return: Returns parser.
+    """
+    tree_sitter_python = clone_repo("https://github.com/tree-sitter/tree-sitter-python")
+    tree_sitter_javascript = clone_repo("https://github.com/tree-sitter/tree-sitter-javascript")
+    tree_sitter_java = clone_repo("https://github.com/tree-sitter/tree-sitter-java")
+    path_to_build = str(Path(f"{Path.cwd().parent}/build/my-languages.so"))
```

Comment:
please make it a constant - you are using it in several places - it's error-prone approach to duplicate code like this